support-frontend-static/README.md

```diff
@@ -0,0 +1,33 @@
+# support-frontend-static
+
+This cloudforms the assets bucket for the assets that
+is used by support-frontend.
+
+# Deploying to PROD (manual)
+
+1. make your changes to cfn
+1. do a PR and get it approved
+1. merge to master
+1. check out master locally
+1. get janus credentials
+1. run ./PROD-deploy.sh
+
+# More information
```

Comment:
Most of this is out of date now isn't it?

Reply:
I thought so, but actually upon reading it it was all still relevant